stl/inc/chrono

```diff
@@ -5528,13 +5584,45 @@ struct formatter<_CHRONO day, _CharT> {
         }
 
         return _Write_aligned(_STD move(_FormatCtx.out()), static_cast<int>(_Stream.view().size()), _Specs,
-            _Align::_Left, [&](_Ctx_iter _Out) { return _Fmt_write(_STD move(_Out), _Stream.view()); });
+            _Align::_Left, [&](auto _Out) { return _Fmt_write(_STD move(_Out), _Stream.view()); });
     }
 
-private:
     _Chrono_format_specs<_CharT> _Specs{};
     bool _No_chrono_specs = false;
 };
+
+template <class _CharT>
+struct formatter<_CHRONO day, _CharT> {
+    typename basic_format_parse_context<_CharT>::iterator parse(basic_format_parse_context<_CharT>& _Parse_ctx) {
+        return _Base._Parse(_Parse_ctx, "de");
+    }
+
+    template <class _FormatContext>
+    typename _FormatContext::iterator format(const _CHRONO day& _Val, _FormatContext& _FormatCtx) {
+        const tm _Time = {.tm_mday = static_cast<int>(static_cast<unsigned int>(_Val))};
+        return _Base._Write(_FormatCtx, _Val, _Time);
+    }
+
+private:
+    _Chrono_formatter<_CharT, false> _Base;

```

Comment:
Nitpick: I'd recommend `_Impl` or something along those lines, since this is not actually a base class. Occurs below.